packages/backstopjs-config/README.md

```diff
@@ -59,24 +65,44 @@ First install the package and its dependencies. This includes headless Chrome, s
 	```
 
 3. Add the npm script to package.json:
-	
+
 	```language:javascript
 	"scripts": {
-		"backstop": "backstop --config=node_modules/@penskemediacorp/backstopjs-config"
+		"backstop": "larva write-json && docker run --rm --network='host' -v $(pwd):/src backstopjs/backstopjs --config=node_modules/@penskemediacorp/backstopjs-config",
+		"backstop:reference": "npm run backstop -- reference",
+		"backstop:test": "npm run backstop -- test && npm run backstop:open-report",
+		"backstop:open-report": "opener ./backstop_data/html_report/index.html",
+		"backstop:pull-docker": "docker pull backstopjs/backstopjs",
 	}
 	```
 
+	Script explanation:
+
+	```bash
+	docker run --rm \
+	-v $(pwd):/src \             # Mount the source code
+	--network='host' \           # Allows docker to access your locally running larva application
+	--user $(id -u):$(id -g) \   # Run as given user (you) - without this, files get saved by root user
+	backstopjs/backstopjs --config=node_modules/@penskemediacorp/backstopjs-config
+	```
+
 ## Running the Tests
 
-After you have added the configuration, it is time to run the tests. 
+After you have added the configuration, it is time to run the tests.
+
+1. Before testing, Larva must be running:
+
+	```
+	npm run larva
+	```
 
-1. You must first generate reference screenshots from a master branch, or other branch that contains the UI you want to test against. When you are on that branch, run the following:
+2. You must first generate reference screenshots from a master branch, or other branch that contains the UI you want to test against. When you are on that branch, run the following:
 	```
 	# Generate reference screenshots
 	npm run backstop -- reference
 	```
-2. Checkout to your feature branch, or the branch that contains changes you want to test for regressions. 
-3. Run the test with: 
+3. Checkout to your feature branch, or the branch that contains changes you want to test for regressions.
+4. Run the test with:
 	```
 	# Run the tests, then open an HTML page with a UI for viewing results.
```

Comment:
We need an additional step about running and updating the tests during development. The workflow is to run the tests when you are ready to open a PR, make sure the diff is correct and there are no regressions, and then `backstop -- approve` to update the screenshot files, and then commit them.

Reply:
thanks, didn't even know about approve before I saw this :) - done